unit-tests/source_tree_test.py

```diff
@@ -61,34 +83,71 @@ def test_extension_visitor(tmp_path: Path):
     (tmp_path / 'dir').mkdir()
     bar_file = tmp_path / 'dir' / 'file.bar'
     bar_file.write_text('Second file in subdirectory')
+    baz_file = tmp_path / 'file.baz'  # Doesn't exist
 
     db = SqliteStateDatabase(tmp_path)
     file_info = FileInfoDatabase(db)
+    clear_tracker()
```

Comment:
Can you use test fixtures to clear the tracker?

Reply:
I think I've now done this using `setup_function`